Win.md

```diff
@@ -0,0 +1,160 @@
+
+Type `create part msr size 128` to create the MSR partition (default size 128 MB, shouldn't be changed; used by some strange Windows things).
+
+Type `create part primary size 128000` to create the main partition (default size = [total space] - 450MB of WinRE - 128MB of MSR - 200MB of EFI. I remember that the size value must be in MB) --> default size = [total space] - 778MB, then type `format quick` to format it and `assign letter c` to assign correctly our drive.
+
+After that we set up the recovery enviroment: type `create part primary` (default size 450, the remaining part of the disk), `set id de94bba4-06d1-4d40-a16a-bfd50179d6ac` to change the type of partition (primary-->WINRE, more infos [here](https://en.wikipedia.org/wiki/GUID_Partition_Table#Partition_type_GUIDs)),`gpt attributes 0x8000000000000001` to hide it,`format quick` to format it in NTFS.
+
+Type `exit` to quit diskpart.
+
+#### Install Windows
+
+Type `dism /get-wiminfo /wimfile:<path>` to get the number of your edition.
+
+Type `dism /apply-image /imagefile:<path> /index:4 /applydir:c:\` to flash the WIM/ESD image to your disk where 4 is changed as you edition number.
+
+Type `bcdboot c:\windows /s q:\` to create the boot files on the EFI disk (Caution! if OpenCore was present, Windows cannot inject your EFI).
+
+Reboot and hope everything work.
+
+### Legacy Windows Installation (broken)
+
+::: details Legacy Users (NOTE: at the moment I didn't find a how to create the mbr bootloader...)
+
+#### Manually MBR partitions
+
+![Legacy Partitions](/images/win-md/partL.png)
+
+Type `diskpart`
+
+Type `list disk` and remember the disk number where you want to install Windows
+
+Type `sel disk W` where W is the number of your disk
+
+Type `clean` to clear the whole disk
+
+We set up the System Reserved Partition: type `create part primary size 100` to create the MSR partition (default size 100) `remove` to remove its letter.
+
+Type `create part primary size` (default size [totalspace]-550 MB. I remember that the size value should be in MB) to create the main partition, then type `format` to format it and `assign letter c` to assign correctly our drive.
+
+Finally we set up the recovery enviroment: type `create part primary` (default size 450, the remaining part of the disk), `set id 27` to change the type of partition (primary-->WINRE),`format` to format it in NTFS and `remove` to remove its letter.
+
+#### Legacy Installation
+
+Type `dism /get-wiminfo /wimfile:<path>` to get the number of your edition.
+
+Type `dism /apply-image /imagefile:<path> /index:4 /applydir:c:\` to flash the WIM/ESD image to your disk where 4 is you edition number.
+
+**TODO**: I have no idea how to create the boot files in the MBR... If you know this, you can help contibute via [PRs](https://github.com/dortania/OpenCore-Multiboot/pulls).
```

Comment:
I dont think MBR should be treated here as OC has DUET which emulates a UEFI environment.

Reply:
I agree, I will leave it as far I can